packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -141,7 +141,12 @@ class CreateCommand extends FlutterCommand {
     printStatus('Wrote $generatedCount files.');
     printStatus('');
 
-    updateXcodeGeneratedProperties(appPath, BuildMode.debug, flx.defaultMainPath);
+    updateXcodeGeneratedProperties(
+      projectPath: appPath,
+      mode: BuildMode.debug,
+      target: flx.defaultMainPath,
+      hasPlugins: false,
```

Comment:
Should be `hasPlugins: generatePlugin,`. The example app for a plugin project does have plugins. :)